fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -374,6 +374,10 @@ protected void updateSchema() {
         if (!MULTITENANT_FEATURE_ENABLED.contains(dbType) && newDb ) {
             populateResourceTypeAndParameterNameTableEntries(null);
         }
+
+        // Let's refresh the procedures and functions.
+        logger.info("Refreshing procedures and functions");
+        updateProcedures();
```

Comment:
I was under the impression that the stored procs (or functions for postgres) should be part of the physical data model already and that they should get "refreshed" as part of the normal applyModel.
Obviously that wasn't working quite right, but now I'm not sure if it was because my understanding was wrong or if a premature commit (the change below) was the root cause.
I'm OK with adding this in to cover us (so for sure the issue will be addressed), but it would be good to know whether applyModel above is working properly or not wrt updating procedures.

Reply:
yes, that's right they should however it doesn't exactly work on postgres, it does on db2.